letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -500,6 +500,7 @@ Bootstrap() {
     echo "You will need to bootstrap, configure virtualenv, and run pip install manually."
     echo "Please see https://letsencrypt.readthedocs.org/en/latest/contributing.html#prerequisites"
```

Comment:
That's outside the scope of this problem this PR is trying to fix (which continuing `certbot-auto` execution after failing to bootstrap, resulting in awful errors when installing/using the Python packages), however, given the size of this PR, I'm fine addressing it here if you prefer.

With that said, I don't think we actually have a good link in this case. The instructions you linked to tell the user to run `certbot-auto` in order to install OS packages (which just failed for them).

Perhaps we should try to put together a list of OS dependencies somewhere, however, this list varies based on your OS.